[2025-03-10 10:52:38,231] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: xkcd_batch_loader.store_xkcd_data 2025-03-10T10:52:37.837330+00:00 [queued]>
[2025-03-10 10:52:38,239] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: xkcd_batch_loader.store_xkcd_data 2025-03-10T10:52:37.837330+00:00 [queued]>
[2025-03-10 10:52:38,239] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2025-03-10 10:52:38,239] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2025-03-10 10:52:38,240] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2025-03-10 10:52:38,261] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_xkcd_data> on 2025-03-10T10:52:37.837330+00:00
[2025-03-10 10:52:38,264] {standard_task_runner.py:52} INFO - Started process 176 to run task
[2025-03-10 10:52:38,265] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'xkcd_batch_loader', 'store_xkcd_data', '2025-03-10T10:52:37.837330+00:00', '--job-id', '74', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/XKCD_batch/dag.py', '--cfg-path', '/tmp/tmpnf8lcjha', '--error-file', '/tmp/tmpnxobx204']
[2025-03-10 10:52:38,268] {standard_task_runner.py:77} INFO - Job 74: Subtask store_xkcd_data
[2025-03-10 10:52:38,444] {logging_mixin.py:104} INFO - Running <TaskInstance: xkcd_batch_loader.store_xkcd_data 2025-03-10T10:52:37.837330+00:00 [running]> on host 4dcdde3a97c5
[2025-03-10 10:52:38,671] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xkcd_batch_loader
AIRFLOW_CTX_TASK_ID=store_xkcd_data
AIRFLOW_CTX_EXECUTION_DATE=2025-03-10T10:52:37.837330+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-10T10:52:37.837330+00:00
[2025-03-10 10:52:38,715] {base.py:78} INFO - Using connection to: id: ***_postgresql. Host: ***, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2025-03-10 10:52:38,747] {base.py:78} INFO - Using connection to: id: ***_postgresql. Host: ***, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2025-03-10 10:52:38,760] {python.py:151} INFO - Done. Returned value was: None
[2025-03-10 10:52:38,785] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=xkcd_batch_loader, task_id=store_xkcd_data, execution_date=20250310T105237, start_date=20250310T105238, end_date=20250310T105238
[2025-03-10 10:52:38,821] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10 10:52:38,839] {local_task_job.py:149} INFO - Task exited with return code 0
